### ISSUE TITLE
Can't open right click menu for Viber v18.2.0.2 (latest)

### ISSUE THREAD

**Issue by xalt7x** [ubuntu/gnome-shell-extension-appindicator#375]
With extension v46 I have Viber icon (it's missing with other extensions and version shipped with Ubuntu 22.04) but it reacts only for the left click. Right click does nothing so I can't normally exit application.
Issue is reproducible with both[ official deb package](https://download.cdn.viber.com/cdn/desktop/Linux/viber.deb) and [Flatpak](https://flathub.org/apps/details/com.viber.Viber). It's probably more GNOME/extensions issue because right click menu works on KDE Plasma.
Fortunately I've saved an old deb package of Viber v13.3.1.22 which still works fine.
P.S. Found similar report [here](https://github.com/flathub/com.viber.Viber/issues/4#issuecomment-1259323409)
P.P.S Workaround for now is to add "dbus-launch" prior to launch command
For deb package:
`dbus-launch /opt/viber/Viber %u`
For Flatpak:
`dbus-launch /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=viber com.viber.Viber`

**Comment by tezvi** [comment]
Thx for deb package instructions. It works.

**Comment by VladimirBelyavsky** [comment]
This is probably a common issue for all Qt-based apps - https://bugreports.qt.io/browse/QTBUG-107727

**Comment by nbulaj** [comment]
`dbus-launch /opt/viber/Viber %u` works for me (restores missed tray icon), but I have the same issue with 3 clicks to open a popup menu :disappointed: ubuntu 22.04

```bash
$ gnome-shell --version
GNOME Shell 42.5, 

$ gnome-extensions --version
42.5
```

**Comment by vladislav-svetailo** [comment]
Linux mint 21.1 - `dbus-launch /opt/viber/Viber %u` works! Same, two right clicks needed, but, it is not a problem.

**Comment by n1kcha** [comment]
After latest 2 updates, right click stop working for me (Fedora 37). More than one right clicks don't work. dbus-launch /opt/viber/Viber %u not made any difference

**Comment by teras** [comment]
> After latest 2 updates, right click stop working for me (Fedora 37). More than one right clicks don't work.` dbus-launch /opt/viber/Viber %u` not made any difference

I have the same problem... ` dbus-launch` makes no difference.

**Comment by GitEin11** [comment]
fixed on 20.3.0.1

**Comment by Tamas-Toth-ebola** [comment]
FYI, it seems that in the last few weeks, it started to work as expected. No 3 clicks, no 'special menu' _(I mean Viber's own overlay type)_. Everything works like with all other applications/GNOME panel extensions.

**Thanks for all the efforts!** Now there is no problem, at least for me. My current Viber version is 20.3.01

**Comment by teras** [comment]
> From my aspect the issue cuould be closed.

It works for me too

**Comment by xalt7x** [comment]
This bug was fixed on Viber 20.3.0.1. Users could either install [official](https://www.viber.com/en/download/) version (AppImage, DEB, RPM) or wait for updated [Flatpak](https://flathub.org/apps/com.viber.Viber) .
